src/repos/UserRepo.ts

```diff
@@ -99,7 +141,26 @@ const appendReadArticle = async (uuid: string, articleID: string): Promise<User>
 };
 
 /**
- * Add a magazine to a user's readMagazines
+ * Add a magazine to a user's readFlyers
+ */
+const appendReadFlyer = async (uuid: string, flyerID: string): Promise<User> => {
+  const user = await UserModel.findOne({ uuid });
+
+  if (!user) return user;
```

Comment:
if `(!user)` is true, then shouldn't the return type be `Promise<User | undefined>`? does this line not give u an error :o

Reply:
what exactly does `user` contain when `(!user)` is true?

Reply:
i checked in graphql playground and it just returns null for every field queried